packages/svelte-urql/src/common.ts

```diff
@@ -0,0 +1,140 @@
+import type { Readable, Writable } from 'svelte/store';
+import type {
+  OperationResult,
+  RequestPolicy,
+  OperationContext,
+  Client,
+} from '@urql/core';
+import { pipe, fromValue, concat, map, makeSubject, onPush } from 'wonka';
+import type { Source } from 'wonka';
+import type { DocumentNode } from 'graphql';
+import type { TypedDocumentNode } from '@graphql-typed-document-node/core';
```

Comment:
```suggestion
import type {
  OperationResult,
  RequestPolicy,
  OperationContext,
  Client,
  TypedDocumentNode,
} from '@urql/core';
import { pipe, fromValue, concat, map, makeSubject, onPush } from 'wonka';
import type { Source } from 'wonka';
import type { DocumentNode } from 'graphql';
```

This is just a small note. We reexport `TypedDocumentNode` from `@urql/core` and use it from there. If there's a case where we'll switch to the main `graphql` package's generics (if it'll add them) or we switch away from this shared package, it's useful for that to be independent of the bindings.

Reply:
thanks!